### DISCUSSION TITLE
How to apply rotation 270 degree of batch list ?

### DISCUSSION THREAD

**Discussion by ANDYSNOWFOX** [mifi/lossless-cut#899] [Support]
I added a list of mp4 to a batch list and I want to rotate 270 degree of all the files (from mobile phone), but I can only change the rotation setting of the one shows up, after click other file and back, there is no more rotation setting saved. I have to do the rotation and export for each file by myself. It is not "batch" for I know. 
It supposed to be changed one setting and automatically applied to all files and just need one export click in the batch.
It must be something wrong with my operation.

I searched on internet and got no hints, so want to check here.

**Comment by mifi** [comment]
Currently the batch function is very simple and it just allows easily adding multiple files and stepping through them using keyboard shortcuts. See progress on the feature here: #89
